src/components/AsyncTable/Controls/Controls.tsx

```diff
@@ -0,0 +1,32 @@
+//- Type Imports
```

Comment:
nit: we should remove these grouping comments as this is one of the outdated practices from `zns-dapp`

```
import React 'react';
import type { FC } from 'react';

import { IconButton } from '../../IconButton';
import { IconList, IconGrid } from './Icons';
import { SearchBar } from './SearchBar';

import styles from './Controls.module.scss';
```

Reply:
it is just copied

Reply:
thanks for your comment for this, will keep from now.

Reply:
Nice thanks, I forgot to remove them 😄